src/ng/parse.js

```diff
@@ -1703,11 +1703,11 @@ function $ParseProvider() {
 
   this.$get = ['$filter', '$sniffer', function($filter, $sniffer) {
     var $parseOptions = {
-          csp: $sniffer.csp,
+          csp: $sniffer.csp && $sniffer.csp.noUnsafeEval,
```

Comment:
Ah good point. Probably an indication that `$sniffer`is doing too much, but that ship has sailed.